packages/xo-web/src/common/render-xo-item.js

```diff
@@ -683,6 +683,18 @@ const xoItemToRender = {
   ),
 
   PCI: props => <Pci {...props} self />,
+
+  schedule: schedule => {
+    const isEnabled = schedule.enabled
+    return (
+      <span>
+        <span>{schedule.name}</span>
```

Comment:
Should we do like in Backup > Overview and fallback to `schedule.cron` if `schedule.name` is empty?

Reply:
We have already discussed it with Florian. Cron can be **very very long**, and would not be suitable in a selector.

Reply:
Makes sense, indeed.
But in the selector, it really doesn't make sense to me to show some "Enabled" and "Disabled" tags alone when the schedule doesn't have a name. And I guess it can happen, especially if the user is used to only have one schedule per job: they might only care about the job's name and never set a schedule name.
So could we maybe show the job's name next to the schedule's name?

```jsx
<span>{schedule.name} ({job.name})</span>
```

Reply:
And to avoid long labels, what do you think of grouping schedules by job?

Reply:
That's even better :) However, let's still fallback to something like `<em>Unnamed schedule</em>` to be more explicit about what is being listed, and so that we don't show the state pill alone in a row. Also what do you think about putting that pill in front so that they're all aligned? The same way we do it to render a `backup`.